feed-integration-webapp/src/test/java/org/bahmni/module/feedintegration/integrationtest/HL7Utils.java

```diff
@@ -0,0 +1,54 @@
+package org.bahmni.module.feedintegration.integrationtest;
+
+import ca.uhn.hl7v2.*;
+import ca.uhn.hl7v2.model.*;
+import ca.uhn.hl7v2.model.v25.message.*;
+import ca.uhn.hl7v2.model.v25.segment.*;
+
+import java.text.*;
+import java.util.*;
+
+public class HL7Utils {
```

Comment:
Is this class being used for CraterAtom feed?

Reply:
Class/Test was not being used so has been removed.